share/picongpu/examples/IncidentField/include/picongpu/param/incidentField.param

```diff
@@ -205,7 +206,7 @@ namespace picongpu
                  *                  field_internal = field_SI / unitField
                  */
                 HINLINE FunctorYMaxIncidentE(float_X const currentStep, float3_64 const unitField)
-                    : FunctorBase(currentStep)
+                    : FunctorBase(currentStep - static_cast<float_X>(T_delaySteps))

```

Comment:
Is `FunctorBase` able to handle negative times?

Reply:
Yes